index.d.ts

```diff
@@ -154,6 +154,7 @@ export interface VerifyOTPProps extends ExtraTextProps {
 
   onEnterCode?: (code: string, clearCode: () => void) => void;
   onPressResend?: (restartTimer: () => void) => void;
+  onCountDownFinished?: () => void;
```

Comment:
Maybe `onCountDown?: (remainingSeconds: number) => void;` is more generic. We can check if `remainingSeconds === 0` to see if it is finished.

What do you guys think? @cychiuae @clarkwkw

Reply:
Make sense to me

Reply:
Also make sense to me

Reply:
Remember to invoke the callback when we set the interval, otherwise for a countdown of 5 seconds we only have the following invocations

```
onCountDown(4);
onCountDown(3);
onCountDown(2);
onCountDown(1);
onCountDown(0);
```

where 

```
onCountDown(5);
```

is missing.